.travis.yml

```diff
@@ -109,6 +109,15 @@ jobs:
     - name: PHP unit tests w/ external-http (7.3, WordPress latest)
       php: "7.3"
       env: WP_VERSION=latest DEV_LIB_ONLY=phpunit                                INSTALL_PWA_PLUGIN=1 PHPUNIT_EXTRA_SUITE=external-http
+      script:
+        - npm run build:js
+        - npm run build:css
+        - source "$DEV_LIB_PATH/travis.script.sh"
+        - |
+          if [[ ! -z "$PHPUNIT_EXTRA_SUITE" ]]; then
+            echo "Running phpunit testsuite $PHPUNIT_EXTRA_SUITE"
+            travis_retry phpunit --testsuite $PHPUNIT_EXTRA_SUITE
```

Comment:
Shouldn't this change rather be made in the existing `script` definition above?

Reply:
It depends on whether you want to have everything be retried, not only the external HTTP tests.

I'd say, for non-external tests, there's no reason to retry as they shouldn't fail for temporary reasons. Retrying would only run all failures always three times and thus slow everything down until we get test failure results...

Reply:
The only value for `PHPUNIT_EXTRA_SUITE` is `external-http` so wouldn't it be limited to external requests?

Couldn't this PR be thus simplified down to just:

```diff
diff --git a/.travis.yml b/.travis.yml
index e70257eca..8e63ae03d 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -50,7 +50,7 @@ script:
   - |
     if [[ ! -z "$PHPUNIT_EXTRA_SUITE" ]]; then
       echo "Running phpunit testsuite $PHPUNIT_EXTRA_SUITE"
-      phpunit --testsuite $PHPUNIT_EXTRA_SUITE
+      travis_retry phpunit --testsuite $PHPUNIT_EXTRA_SUITE
     fi
 
 after_script:
```

Reply:
Oh, I get it now. Sorry, I didn't properly read the conditional check before. You're right, I'll change it.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/4298/commits/200699501ffaeeba324fc21e030fb2159f197738